clonecatalog.py

```diff
@@ -305,6 +308,9 @@ def _get_or_create_transfer(
         )
         target_securable_name = re.findall("[^.]+$", target_securable_full_name)[0]
         try:
+            if securable_type == catalog.SecurableType.TABLE and self.force_deep_clone:
+                raise DatabricksError("Force deep clone for table")
+
```

Comment:
Could you elaborate on this? what is it doing?

Reply:
The deep clone occurs in the original code when an exception is raised if the table doesnt exists, using the .get (table) method in the sdk. In this case, just before the .get, I will raise an exception to force the deep clone execution.
Look at the try/catch as a block where deep clone executes or not. We don't use a method of if/elses in the original code.